utils/uuid.js

```diff
@@ -1,10 +1,10 @@
-import uuid from 'uuidv4';
+import { fromString } from 'uuidv4';
 
 /**
  *
  * @param {String} str
  */
-const generateFromString = (str) => uuid.fromString(str);
+const generateFromString = (str) => fromString(str);
 
 export default {
   generateFromString,
```

Comment:
Useless utils. Please call `uuid.fromString` directly and remove this utils.

Reply:
Solved: https://github.com/EasyEventV2/ee2_api/pull/9/commits/a3731140fdf0c033a5cab01e3222afc12fde85b3